mrtt-ui/src/validation/multiSelectWithOther.js

```diff
@@ -14,15 +14,10 @@ export const otherIsCheckedButInputIsEmptyValidation = (schema) =>
   })
 
 export const multiselectWithOtherValidation = yup.object({
-  selectedValues: yup
-    .array()
-    .default([])
-    .when('isOtherChecked', {
-      is: false,
-      then: (schema) =>
```

Comment:
I got rid of the min requirement here. It was causing some frustrating issues in section 5, where checkbox groups would be hidden, unless, some other field was selected, which would then cause a hidden error and wouldn't let you submit the form. I think we should get rid of most of the required yup validation for now, given that we don't actually know what is required, but also because users are able to submit incomplete forms and come back to them at a later time.